src/GeoJson/BoundingBox.php

```diff
@@ -1,48 +1,55 @@
 <?php
 
+declare(strict_types=1);
+
 namespace GeoJson;
 
 use GeoJson\Exception\UnserializationException;
+use InvalidArgumentException;
+use JsonSerializable;
+
+use function count;
+use function is_int;
+use function is_float;
+use function is_array;
 
 /**
  * BoundingBox object.
  *
  * @see http://www.geojson.org/geojson-spec.html#bounding-boxes
  * @since 1.0
  */
-class BoundingBox implements \JsonSerializable, JsonUnserializable
+class BoundingBox implements JsonSerializable, JsonUnserializable
 {
     /**
-     * @var array
+     * @var array<float|int>
      */
-    protected $bounds;
+    protected array $bounds;
 
     /**
-     * Constructor.
-     *
-     * @param float[] $bounds
+     * @param array<float|int> $bounds
```

Comment:
Thanks for catching this. I see it agrees with "BoundingBox values must be integers or floats" below.